### PR TITLE
complete the 404 page

### PR DESCRIPTION
**Pull request by mrcongliu**
Add Page404 with issues:
1. Font-family may not be exactly same as Mockup Page.
2. 404Icon(emoji) SVG disappears on mobile screen.
Trello:
https://trello.com/c/NKQe9qeB/175-translate-404-page-mockup-to-code

### CODE REVIEW

File: src/pages/Page404/Page404.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@media all and (min-width: 360px) {
```
Comment (ArnasDickus) [inline src/pages/Page404/Page404.css]: Smallest phones are 320px. You added min-width: 360px. Everything below 360px will lose styling and text color.
404-error

File: src/components/UI/Button/Button.module.css [hunk 2]
```diff
@@ -63,6 +63,11 @@
     font-size: 1rem;
 }
 
+.Page404 {
+    padding: 7px 50px;
```
Comment (ArnasDickus) [inline src/components/UI/Button/Button.module.css]: Rem is usually a better measurement than px due to accessibility. Change 50px to rem

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, thanks for pull request. Everything looks good apart from 3 minor things.

1) Your code is not formatted. To format code run `npm run format`, don't worry if other files are going to be prettified as well.

**Comment by mrcongliu** [comment]
All three issues of page404 have been fixed.

**Comment by ArnasDickus** [comment]
Hello, @CongLiu-CN I reopened this pull request. Why did you close it?

**Review by ArnasDickus — APPROVED** [review]

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, @CongLiu-CN  on the mobile view way too much padding. Either adjust it with media queries or instead of padding use `max-width` and `margin: auto`
404-page-2

**Comment by mrcongliu** [comment]
Sorry I closed this PR, because I thought I need to open another one after I made git push commands. I already fix it by adding max-width of the container. Thanks.

**Review by ArnasDickus — APPROVED** [review]

**Comment by ArnasDickus** [comment]
@CongLiu-CN  Regarding the svg icon, disappearing navbar css is conflicting with it. So I'm going to approve this PR and add another task as a bug report.

**Pull request merged**